### ISSUE TITLE
Scopes: Adds scope to control if a user can unassign another user

### ISSUE THREAD

**Issue by jpye-finch** [opencrvs/opencrvs-core#7431]
###Description
Currently a registrar can unassign another user from a record. We need to add a scope so that this permission can be added to the roles defined in country config

This ticket is to:
- add scope `record.unassign-others`
- when a user has this scope then they can unassign a user who is currently assigned to the record
- remove the current approach to unassigning the user by clicking on the user's profile image
- If another user is assigned to the record then it shows their name and location in the Action menu

![Image](https://github.com/user-attachments/assets/a9a50769-ef94-4928-8167-df31b0661d19)

### ACs
GIVEN a record is assigned to another user
THEN show their name and location in the Action menu option list

GIVEN a record is assigned to another user
WHEN a user has the scope `record.unassign-others`
THEN show the action option 'Unassign'

GIVEN a user has the scope `record.unassign-others`
WHEN they click Unassign
THEN show the confirmation model

GIVEN a record is assigned to another user
WHEN a user clicks their user's profile image
THEN do nothing

GIVEN a record is assigned to me
THEN show the action option 'Unassign'

### Dev tasks

**Comment by Zangetsu101** [comment]
@jpye-finch There is a bit an edge case here when a record is assigned to a user and they log in from a different device. The download button will be at the blue download state & the unassign action will also be available

**Comment by jpye-finch** [comment]
Yes properly supporting using a multiple devices is not prioritised yet. To address some of the odities. ie. when they log in we check if they have assigned records on another device and then prompt them to "sync" with this device.

Users are often assigned to one device

**Comment by Zangetsu101** [comment]
We have identified a bug regarding the "Unassign" action which requires us to decouple the `assignment` data from the `workqueues` in client

**Comment by Zangetsu101** [comment]
We will need to create a separate redux slice for storing assignment data. All the places in client will read assignment related data from it. `Workqueu Update`, `Assign` & `Unassign` actions should update the redux state

**Comment by SyedaAfrida** [comment]
This ticket is fixed
